systemtest/src/test/java/io/strimzi/systemtest/bridge/HttpBridgeCorsST.java

```diff
@@ -68,7 +64,7 @@ void testCorsOriginAllowed() {
 
         String url = bridgeUrl + "/consumers/" + groupId + "/instances/" + kafkaBridgeUser + "/subscription";
         String headers = BridgeUtils.addHeadersToString(additionalHeaders, Constants.KAFKA_BRIDGE_JSON_JSON);
-        String response = cmdKubeClient().namespace(namespace).execInPod(scraperPodName, "/bin/bash", "-c", BridgeUtils.buildCurlCommand(HttpMethod.OPTIONS, url, headers, "")).out().trim();
+        String response = cmdKubeClient().namespace(clusterOperator.getDeploymentNamespace()).execInPod(scraperPodName, "/bin/bash", "-c", BridgeUtils.buildCurlCommand(HttpMethod.OPTIONS, url, headers, "")).out().trim();
```

Comment:
I don't like the usage of `clusterOperator.getDeploymentNamespace()` in actions that are not related to CO. Isn't there any better way?

Reply:
well in @ParallelTest or @IsolatedTest where components are deployed in the same namespace where the operator is I think the okayish way to do it is this way. But this could definitely change if we decide to use two namespace (for operator and components)